README.adoc

```diff
@@ -19,12 +19,15 @@
 :page-related-guides: ['microprofile-health', 'kubernetes-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-= Probing microservices running in Kubernetes
+:page-seo-title: Checking the health of microservices using Kubernetes readiness and liveness probes
+:page-seo-description: How to check microservice health status using MicroProfile Health with Kubernetes liveness probes and readiness probes
```

Comment:
Do we want to change page-description to "Learn how to check the health of microservices on Kuberenetes by setting up readiness and liveness probes to inspect MicroProfile Health Check endpoints"?

May want to double check how it looks on the All Guides page i.e. in the card for the guide first.